cmssw-pr-test-config

```diff
@@ -2,4 +2,4 @@ PR_TEST_MATRIX_EXTRAS=1306.0,101.0,9.0,25202.0,10224.0,250202.181
 PR_TEST_MATRIX_EXTRAS_GPU=11634.506,11634.512,11634.522
 PR_TEST_MATRIX_EXTRAS_PROFILING=39634.21,21034.21,11834.21,136.889
 PR_TEST_MATRIX_EXTRAS_HIGH_STATS=35034.0
-PR_TEST_MATRIX_EXTRAS_NANO=2500.0
+PR_TEST_MATRIX_EXTRAS_NANO=2500.0,2500.001,2500.1,2500.101,2500.11,2500.12,2500.2,2500.21,2500.3,2500.301,2500.302,2500.31,2500.311,2500.312,2500.32,2500.321,2500.322,2500.33,2500.331,2500.332,2500.4,2500.401,2500.5,2500.501,2500.51,2500.511,2500.6,2500.601
```

Comment:
no way to have this being a wild-card `2500.*` ?
the nano workflows are done dynamically ; maybe I need to make them static

Reply:
no wild card does not work. It think static wf are better which can easily match entires in `matrix_RE.txt` otherwise we have to keep changing `matrix_RE.txt` for every change in workflow number